app/assets/javascripts/modules/form-validation.js

```diff
@@ -1,198 +1,249 @@
 'use strict'
+
 var formValidation = function () {
-  var form = $('form#card-details'),
-    formInputs = form.find('input'),
-    countryAutocomplete = form.find('.autocomplete__input'),
-    countrySelect = form.find('#address-country'),
-    postcodeInput = form.find('#address-postcode'),
-    cardInput = form.find('#card-no'),
-    errorSummary = $('.govuk-error-summary'),
-    logger = { info: function () {} }, // replace with console to see output
-    // window.card comes from the view
-    chargeValidations = module.chargeValidation(
-      i18n.chargeController.fieldErrors,
-      logger,
-      $.extend({}, window.Card)
-    ),
-    required = chargeValidations.required
+  var form = document.getElementById('card-details')
+  var formInputs = form.querySelectorAll('input')
+  var countryAutocomplete = document.getElementsByClassName('autocomplete__input')[0]
+  var countrySelect = document.getElementById('address-country')
+  var postcodeInput = document.getElementById('address-postcode')
+  var cardInput = document.getElementById('card-no')
+  var errorSummary = document.getElementsByClassName('govuk-error-summary')[0]
+  var logger = { info: function () { } }// replace with console to see output
+  // window.card comes from the view
+  var chargeValidations = module.chargeValidation(
+    i18n.chargeController.fieldErrors,
+    logger,
+    window.Card
```

Comment:
In the previous code we have `$.extend({}, window.Card)` which i would assume is to send an object if window.Card is not defined or potentially it is cloning window.Card to avoid mutations to the original window.Card instance?

Reply:
Hmmm yeah, `window.Card` comes from here — https://github.com/alphagov/pay-frontend/blob/1029efd307e4ef4e5b225607cb6d55164cc2fcf0/app/views/includes/scripts.njk#L13-L17 so at worst it’s going to defined but empty. Which is unlikely, but does fail gracefully.

Reply:
Are you sure the extension wasnt done to avoid mutating the original window.Card? Seems odd that someone would write this code otherwise.